packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -14,6 +14,15 @@ export interface IRefreshSummaryResult {
 	isSummaryNewer: boolean;
 }
 
+export interface IStartSummaryResult {
```

Comment:
>IStartSummaryResult [](http://example.com/codeflow?start=17&length=19)

Is this and RefreshSummaryResult not similar to SubmitSummaryResults in the aspect of the purpose? I see SubmitSummaryResult is defined elsewhere while these two live in a Utils file

Reply:
I put this here as `ValidateSummaryResult` is also in this file.

Reply:
These interfaces are internal to the `container-runtime` whereas the `SubmitSummaryResult` to my understanding is public. We can change the internal interfaces way more easily as they aren't exposed to the customer